bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoStatusCodes.java

```diff
@@ -54,4 +56,36 @@
         ZOOM_MODE.put("11", "1/3");
         ZOOM_MODE.put("12", "1/4");
     }
+
+    // map to lookup disc type
+    public static final Map<String, String> DISC_TYPE = new HashMap<>();
+    static {
```

Comment:
No chance to use Map.of? 
Maybe too many entries...

Reply:
Yes this constructor allows a max of 10 entries. If more than that, Map.ofEntries would have to be used.

Reply:
So I let you change the code.

Reply:
done